playbooks/tasks/portal-role-task-mongo-replicaset-get-status-subtask.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+# Find primary node in the MongoDB replicaset (subtask to loop over)
+# TODO: This ansible playbook task, should be moved to portal role
+
+# Notes:
+# - This task file is expected to be called in a loop over MongoDB replicaset
+#   member hosts with loop item variable {{ host_item }}.
+# - These tasks should be generally delegated to {{ host_item }} host, not run
+#   on the current Ansible host that we are handling.
+
+# Search for MongoDB primary host until you find it.
```

Comment:
If I understand it correctly, this subtask doesn't fetch status, it tries to find the primary node. Maybe we should rename it.

Reply:
It tries to find a primary node and checks if there is at least one replicaset node online.
More things (number of voting members etc.) might be added later during finishing replicaset setup automation.